sunpy/visualization/animator/wcs.py

```diff
@@ -299,10 +310,28 @@ def plot_start_image_2d(self, ax):
 
         return im
 
+    def get_2d_plot_limts(self):
```

Comment:
Is this misspelling of "limits" intentional?

Reply:
Also, this should be a private method

Reply:
Misspelling was not intentional, didn't check that, sorry.